pkg/resource/certificate/hooks.go

```diff
@@ -33,6 +35,10 @@ func (rm *resourceManager) writeCertificateToSecret(
 	input.CertificateAuthorityArn = &caARN
 	resp, err := rm.sdkapi.GetCertificateWithContext(ctx, input)
 	rm.metrics.RecordAPICall("READ_ONE", "GetCertificate", err)
+	for err != nil && strings.HasPrefix(err.Error(), "RequestInProgressException") {
+		time.Sleep(1 * time.Second)
+		resp, err = rm.sdkapi.GetCertificateWithContext(ctx, input)
+	}
```

Comment:
We should not sleep/loop here. We need to requeue. Checkout: https://github.com/aws-controllers-k8s/runtime/blob/main/pkg/requeue/requeue.go#L40-L53